app/build.gradle

```diff
@@ -19,11 +18,11 @@ android {
     }
 }
 
-
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])	
-    testCompile 'junit:junit:4.12'	
-    compile 'com.android.support:appcompat-v7:23.3.0'	
-    compile 'com.android.support:support-v4:23.3.0'
-    compile 'com.android.support:design:23.3.0'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
compile is deprecated in Android so the new term is implementation. So make sure in any of your future projects change them to implementation  from compile